stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -51,4 +95,35 @@ private static String getIdAttribute(View element) {
   public View getViewForHighlighting(Object element) {
     return (View)element;
   }
+
+  private class ElementContext {
+    private View mElement;
+
+    public void hook(View element) {
+      mElement = Util.throwIfNull(element);
+    }
+
+    public void unhook() {
+      if (mElement != null) {
+        mElement = null;
+      }
+    }
+
+    private void setAttributeAsText(Map<String, String> attributeToValueMap) {
+      for (Map.Entry<String, String> entry : attributeToValueMap.entrySet()) {
+        String methodName = "set" + capitalize(entry.getKey());
+        String propertyValue = entry.getValue();
+        mMethodInvoker.invoke(mElement, methodName, propertyValue);
+      }
+    }
+
+    private String capitalize(String str) {
+      if (str == null || str.length() == 0) {
```

Comment:
we could also check to see if the first character is already capitalized? probably won't gain us much in practice, but just seems like an obvious optimization.